lcls_tools/common/pydm_tools/magnet.py

```diff
@@ -0,0 +1,41 @@
+from functools import partial
+from pydm import Display
+
+from lcls_tools.superconducting import scLinac, scLinacUtils as utils
+
+
+class MagnetScreen(Display):
+    def __init__(self, parent=None, args=None):
+        super().__init__(parent, args)
+
+    def ui_filename(self):
+        return 'magnet_template.ui'
+
+    def connectSignals(self, magnet: scLinac.Magnet):
```

Comment:
Is this sc specific? 'scLinac.Magnet'?

Reply:
Ah yeah I wasn't sure how to write this annotation considering that a file for NC magnets doesn't exist yet, but both types will be of type "Magnet" and have the same parameters

Reply:
Should we take a stab at a general magnet file that lives in the pydm dir?